### PR TITLE
🧼 add "not found" component

### LINKED ISSUE #290: 404 component
We should make a 404 (page not found) component with a helpful message if a user visits an invalid url.

To reproduce visit https://upswyng.netlify.com/WhateverTheHeckYouWant

## What to Include

Heading: Page not Found

Body:

Whoops! We're having trouble finding that page. Have something specific you're looking for?

[insert search component used on home page]

You can also go back home(linked to home page) to browse.

### PR DESCRIPTION
**Pull request by jollyjerr**
This PR closes #290

## What does this PR do?

Adds a "not found" component that renders whenever a faulty URL is entered.

Screen Shot 2020-03-26 at 10 48 43 PM

### CODE REVIEW

File: packages/upswyng-web/src/components/NotFound.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  return (
+    <Container>
+      <PageBanner color={colors.orangeDark} text={"Page not Found"} />
+      <Typography variant="h3">
+        Whoops! We&apos;re having trouble finding that page.
+      </Typography>
+      <div className={classes.marginTop}>
+        <SearchForm />
+      </div>
+      <Typography className={classes.marginTop}>
```
Comment (jacobvenable) [inline packages/upswyng-web/src/components/NotFound.tsx]: For positioning these equally, you could use MUIs [<Grid/>](https://material-ui.com/components/grid/#grid) instead. Via spacing you can choose how much space is between them:

```jsx
<Grid container direction="column" spacing={4}>
   <Grid item>
      whoops!
   </Grid>
   <Grid item>
      <SearchForm />
   </Grid>
</Grid>
```

File: packages/upswyng-web/src/components/NotFound.tsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+import SearchForm from "./SearchForm";
+import Typography from "@material-ui/core/Typography";
+import { colors } from "../App.styles";
+import createStyles from "@material-ui/core/styles/createStyles";
+import makeStyles from "@material-ui/core/styles/makeStyles";
+
+const useStyles = makeStyles(() =>
+  createStyles({
+    marginTop: {
+      marginTop: "3.5rem",
```
Comment (jacobvenable) [inline packages/upswyng-web/src/components/NotFound.tsx]: Whether you go with using `<Grid>` or not, go ahead and halve this 😄

File: packages/upswyng-web/src/components/NotFound.tsx [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    },
+  })
+);
+
+const NotFound = () => {
+  const classes = useStyles();
+  return (
+    <Container>
+      <PageBanner color={colors.orangeDark} text={"Page not Found"} />
+      <Typography variant="h3">
```
Comment (jacobvenable) [inline packages/upswyng-web/src/components/NotFound.tsx]:
```suggestion
      <Typography component="p" variant="h3">
```
Good with the heading styles but want it to render as a paragraph for semantic markup.

File: packages/upswyng-web/src/components/NotFound.tsx (hunk 1 above)
Comment (jollyjerr) [reply packages/upswyng-web/src/components/NotFound.tsx]: ❤️grids

**Pull request merged**